rosbag2_transport/include/rosbag2_transport/record_options.hpp

```diff
@@ -58,7 +58,9 @@ template<>
 struct ROSBAG2_TRANSPORT_PUBLIC convert<rosbag2_transport::RecordOptions>
 {
   static Node encode(const rosbag2_transport::RecordOptions & storage_options);
-  static bool decode(const Node & node, rosbag2_transport::RecordOptions & storage_options);
+  static bool decode(
+    const Node & node, rosbag2_transport::RecordOptions & storage_options,
+    int version);
```

Comment:
```suggestion
    const Node & node, rosbag2_transport::RecordOptions & storage_options, int version);
```
Nitpick. Could be in one line.